inference-engine/samples/benchmark_app/utils.cpp

```diff
@@ -113,8 +113,6 @@ std::vector<std::string> parseDevices(const std::string& device_string) {
     if ((comma_separated_devices == "MULTI") || (comma_separated_devices == "HETERO"))
         return std::vector<std::string>();
     auto devices = split(comma_separated_devices, ',');
-    for (auto& device : devices)
-        device = device.substr(0, device.find_first_of(".("));
```

Comment:
The python benchmark_app tool should be updated accordingly. 
Maybe also there is a sense to update README with notes when device id is supported in the parameter.